app/views/clients/_show_navigation.html.slim

```diff
@@ -1,10 +1,9 @@
-
 nav.navbar.section-navigation
   ul.list-inline
-    li= button_link t_action(:edit), edit_client_path(@client)
-    - if policy(Journal).index?
-      li= button_link 'Journal', polymorphic_path([@client, Journal])
+    li= button_link t_action(:edit), edit_client_path(@client), 'success'
+    li= button_link t_action(:print), client_path(@client, print: true)
     - if policy(Client).destroy?
       li= link_to t_action(:destroy), @client, confirm_deleting(@client, 'btn btn-default')
+    - if policy(Journal).index?
+      li= button_link 'Journal', polymorphic_path([@client, Journal])
```

Comment:
is it possible to align the journal button to the right? (since they asked for it to be on the right side)

Reply:
Yes, I didn't realize exactly what they meant. Fixed.